ui/src/components/DigitalIdentityDetailPage.jsx

```diff
@@ -0,0 +1,97 @@
+import { Button, Input } from 'antd'
+import React from 'react'
+import { useParams } from 'react-router-dom'
+
+function DigitalIdentityDetailView() {
+    const [displayName, setDisplayName] = React.useState('')
+    const { selectedId } = useParams()
+
+    const CardData = [
+        {
+            claim_name: "Base KYC",
+            claim_expiration_date: "30/07/2023",
+            issue_date: "09/07/2023",
+            claim_hash: "880bf326b5f662ba480ff48dc514631e",
+            txn_hash: "0x5A55456454688474656454DDBVS"
+        },
+        {
+            claim_name: "Accredited Investor",
+            claim_expiration_date: "30/07/2023",
+            issue_date: "09/07/2023",
+            claim_hash: "980bf326b5f662ba480ff48dc661188",
+            txn_hash: "0x6A55456454688474656454DDRTQQA"
+        }
+    ]
+
+    return (
+        <div >
+            <p className='text-xl p-6'>Identity Details</p>
+            <hr></hr>
+            <div className='p-6 max-[500px]:px-4 flex flex-col gap-4'>
+                <div>
+                    <p>Identity display name *</p>
+                    <div className='mt-3 relative w-full flex'>
+                        <Input className='w-full p-2 text-xl' placeholder='Base KYC' type='text' maxLength={32} onChange={(e) => setDisplayName(e.target.value)} />
+                        <p className='absolute right-5 top-2'>{displayName.length}/32</p>
+                    </div>
+                </div>
+                <div className='border rounded-xl p-6 bg-white flex flex-col gap-3 '>
+                    <p>Investor Wallet Address</p>
+                    <p className='font-light text-4xl max-[500px]:text-base text-gray-300'>0x5555aaadwerrasdw553</p>
+                    <p>Investor KYC ID Provider Account Number</p>
+                    <p className='font-light text-4xl max-[500px]:text-base text-gray-300'>007</p>
+                </div>
+                <p>Active Claims</p>
+                {CardData.map((item) => {
+                    return (
+                        <ClaimCard data={item} />
+                    )
+                })}
+            </div>
+            <div className='flex justify-end max-[500px]:justify-center'>
+                <Button className='max-[500px]:w-[50%] rounded-none my-6 mr-6 h-11 px-10 bg-[#9952b3] text-white'>Back</Button>
+            </div>
+        </div>
+    )
+}
+
+export const ClaimCard = ({ data }) => {
```

Comment:
why is this being exported? If it is being used in other pages, please make it it's own component. Also, can you move it above the main page component if it is staying local? I imagine we will use it in the edit confirmation view, too, so probably best to jut move it out into it's own component now.

Reply:
Updated